en/arrays.texy

```diff
@@ -129,4 +129,101 @@ Arrays::renameKey($array, 'first', 'renamed');
 // $array = ['renamed' => 10, 'second' => 20];
 \--
 
-{{composer: nette/utils}}
+
+flatten(array $array, bool $preserveKeys = false): array .[method]
+------------------------------------------------------------------
+
+Transform multidimensional array to flat array.
+
+/--php
+$array = Arrays::flatten([
+	2 => ['a', ['b']],
+	4 => ['c', 'd'],
+	'e',
+]);
+// $array = ['a', 'b', 'c', 'd', 'e'];
+\--
+
+
+isList(array $array): bool .[method]
+------------------------------------
+
+Check if the array is indexed by numeric keys in descending series from zero.
+
+/--php
+Arrays::isList(['a', 'b', 'c'])); // true
+Arrays::isList([4 => 1, 2, 3])); // false
+\--
```

Comment:
Can you please add third example `Arrays::isList(['a' => 1, 'b' => 2])); // false`?